include/ozo/transaction.h

```diff
@@ -75,4 +101,50 @@ struct rollback_op {
 
 constexpr rollback_op rollback;
 
+/**
+ * @ingroup group-connection-functions
+ * @brief Retrieve's a transactions isolation level
+ *
+ * The isolation level can either be an instance of one of the child types of ozo::isolation_level or
+ * `ozo::none` if no level was explicitly set.
+ *
+ * @param transaction transaction to get the isolation level from
+ * @return isolation level or none
```

Comment:
It is better to write something like this

```
@return ozo::isolation_level --- if the isolation level has been set
@return ozo::none --- if no isolation level has been specified for the transaction
```

In this case, necessary links should be generated by Doxygen.